src/content/self-hosted/administration/configuration.mdx

```diff
@@ -1266,3 +1266,29 @@ ingress-nginx:
 ```
 
 The full list of values is [available here](https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/values.yaml).
+
+#### Okteto ingress-nginx default values
+
+Okteto Self Hosted applies some defaults to ingress-nginx, which can be checked before overriding them with the following command:
+
+```
+helm get values okteto/okteto --jsonpath '{.ingress-nginx}'
+```
+
+#### Use a different ingressClass
+
+Okteto Self Hosted uses the following ingressClass value by default in both, control plane and dev user ingresses: `okteto-nginx`.
+
+To use a different value, use the following snippet:
```

Comment:
Why would you need to do this? Could we include some info on the use case or pain point this solves?

Reply:
I think using a different ingress class/ingress controller requires its own doc. It also requires configuring other things to make it work. I would remove this section for now